gn/build/config/compiler/BUILD.gn

```diff
@@ -136,15 +136,19 @@ config("optimize_default") {
 config("disabled_warnings") {
   cflags = [
     "-Wno-deprecated-declarations",
-    "-Wno-unknown-warning-option",
     "-Wno-missing-field-initializers",
-    "-Wno-unused-but-set-variable",
     "-Wno-unused-variable",
   ]
-  cflags_cc = [
-    "-Wno-non-virtual-dtor",
-    "-Wno-deprecated-copy",
-  ]
+  if (!_is_embedded_arm) {
```

Comment:
This change breaks the build.

Reply:
Sorry, which build does it break? I had to add it to _un_break android build that's one of the checks.

Reply:
I think you can drop all of these warnings changes completely.

Reply:
done